andela/boondocks-bn-backend
# #169257389 Setup dotenv

#### What does this PR do?
Set up dotenv to load environment variables from a .env file into process.env

#### Description of Task to be completed?
- [x] Pre-load dotenv when running start or build scripts from package.json
- [x]  Create .env.example to work like blueprint for actual .env file

#### How should this be manually tested?
- using an editor of your choice, create a '.env' file in the project's root folder and input the following line of code: `NODE_ENV=development`
- while still in the root folder, create a test javascript file named 'testit.js' with the following code:
```
const dotenv = require('dotenv');
dotenv.config(); 
const aString = process.env.NODE_ENV;
console.log(aString);
```
- navigate to the root folder of the project using Terminal
- execute the file above by running the following command on your terminal `node ./testit.js`
- you should see the string 'development' printed to the console
- discard the 'testit.js' file once you are done

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#169257389](https://www.pivotaltracker.com/n/projects/2407417/stories/169257389)

#### Screenshots 
- N/A
